Extensions/xkit_preferences.js

```diff
@@ -614,6 +614,11 @@ XKit.extensions.xkit_preferences = new Object({
 		$("#xkit-control-panel-shadow").click(function() {
 			XKit.extensions.xkit_preferences.close();
 		});
+		$(document).keydown(function(e) {
```

Comment:
This event listener should be attached in the `run` function (line 18). The `open` function potentially gets called several times per page which will cause this to unnecessarily duplicate event listeners.

It also should check for the existence of the `#xkit-control-panel` element before calling `close()` as `running`  is true when the `xkit_preferences` dialog isn't open.

An alternative fix would be to remove this event listener in the `close` function